Enriched definition

PavlosIsaris opened the pull request:
Added info about review results and common practices.

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

PavlosIsaris committed:
Enriched definition

Added info about review results and common practices.

lynxlynxlynx commented on client/src/pages/guide/english/working-in-tech/code-reviews/index.md:
```diff
@@ -12,3 +12,9 @@ grow as a developer.
 and be professional when pointing out improvements or errors in the code of your fellow
 team members.
+
+Code Review have been proven to improve quality, modularization and reduce error-proneness in a software project.
```
Please rewrite the first sentence, as it's awkward and has some grammar mistakes.

PavlosIsaris committed:
Fixed grammar mistakes

Clarified the purpose of code reviews, re-written a sentence to improve it's clarity.

lynxlynxlynx commented:
LGTM
